pendulum/generate_table.py

```diff
@@ -13,7 +13,7 @@ def generate_table(out):
 
     # IPOPT
     ocp = prepare_ocp(biorbd_model_path=model_path)
-    opts = {"linear_solver": "ma57"}
+    opts = {"linear_solver": "ma57",  "print_level": 0}
```

Comment:
`ma57` and `print_level=0` gives highly misleading error message. I fear this will lead to think that the examples are broken (instead of their installation is wrong...)